editor/osm_auth.hpp

```diff
@@ -0,0 +1,122 @@
+#pragma once
+
+#include "std/string.hpp"
+#include "std/utility.hpp"
+
+namespace osm
+{
+
+using TKeySecret = pair<string /*key*/, string /*secret*/>;
```

Comment:
If it's a cross-class type, why not to create a struct KeySecret { string m_key; string m_secret; } instead of type alias? You will be able to create a fancy DebugPrint and other stuff then.